numpy/core/src/umath/loops.c.src

```diff
@@ -455,7 +455,9 @@ BOOL_@kind@(char **args, npy_intp const *dimensions, npy_intp const *steps, void
 #define @TYPE@_floor_divide @TYPE@_divide
 #define @TYPE@_floor_divide_indexed @TYPE@_divide_indexed
 #define @TYPE@_fmax @TYPE@_maximum
+#define @TYPE@_fmax_indexed @TYPE@_maximum_indexed
 #define @TYPE@_fmin @TYPE@_minimum
+#define @TYPE@_fmin_indexed @TYPE@_minimum_indexed
```

Comment:
These are just for the integer loops. Is float only done in the `loops_minmax_dispatch`?

Reply:
Yes. I made sure the indexed loops were used in the `test_ufuncs.test_ufunc_at_inner_loops` test for `[fmax, fmin, maxmimum, minimum] * [relevant dtypes]`. I used a hack that I then removed to tell me which loop was used, and could `assert "trivial_at_loop" used`